mage_ai/settings/__init__.py

```diff
@@ -27,10 +29,15 @@ def is_disable_pipeline_edit_access():
 
 DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
 
+#------------------------- DISABLE TERMINAL ----------------------#'
+
+DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
+
 # ----------------- Authentication settings ----------------
 REQUIRE_USER_AUTHENTICATION = \
     os.getenv('REQUIRE_USER_AUTHENTICATION', 'False').lower() in ('true', '1', 't')
 AUTHENTICATION_MODE = os.getenv('AUTHENTICATION_MODE', 'LOCAL')
+TOKEN_PERIOD = os.getenv('TOKEN_PERIOD', '30d').lower()
```

Comment:
i think it would be better to make this a number value which represents the amount of time in seconds

Reply:
let's name it something a little more descriptive like: `MAGE_ACCESS_TOKEN_EXPIRY_TIME`

Reply:
Add changes according to suggestions